### DISCUSSION TITLE
Add tag to display name for leavers

### DISCUSSION THREAD

**Discussion by danmansfield** [KelvinTegelaar/CIPP#238] [Ideas]
Our standard process is to prepend {Leaver] to the display name when offboarding users.
It would be great to have that field available and be able to set as the default so the tech hasn't even got to type it in.

**Comment by KelvinTegelaar** [comment]
We've had this request before and it's too "one-offish" to implement, each MSP wants something else, one want's it prepended, other appended. It would really be hard to please everyone so we don't have this currently. feel free to make a PR yourself to create it though.
